infra/base-images/base-builder/jcc/jcc.go

```diff
@@ -42,6 +42,22 @@ func CopyFile(src string, dst string) {
 	}
 }
 
+func RemoveFailureCausingFlags(cmd []string) []string {
+    // Skip arguments that convert warnings to errors or make the command fail.
+    var newCmd []string
+    for _, arg := range cmd {
+        // Skip arguments that convert warnings to errors
+		if strings.HasPrefix(arg, "-Werror") ||
+			arg == "-pedantic-errors" ||
+			arg == "-Wfatal-errors" ||
+			arg == "-Weverything" {
```

Comment:
Should this be just `-Werror` and `-Wfatal-errors` instead? The other two (at least `-Weverything`) should not result in a compilation error when they trigger, afaik

Reply:
Thanks Mihai!
Yes, I should remove `-Weverything`. I was under the (incorrect) impression that too many warnings would somehow cause the command to fail.

I add `-pedantic-errors` here because I reckon it will generate errors for any code that does not strictly adhere to the ISO C or ISO C++ standards.
Please let me know if I misinterpreted it : )